SQL/Archive/2018-03-10-permission_roles.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE `role` (
+ `RoleID` INTEGER unsigned NOT NULL AUTO_INCREMENT,
+ `Name` varchar(255),
+ `Label` varchar(255),
+ PRIMARY KEY (`RoleID`),
+ UNIQUE KEY `UK_Name` (`Name`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
aren't these supposed to be utf8_mb4 according to our current standards?

Reply:
are they ?? I havnt seen any table creations with `_mb4` specified

Reply:
have there been any tables created since the standards went into effect?